fastchat/serve/monitor/clean_battle_data.py

```diff
@@ -35,33 +36,23 @@
     "lamda",
     "google",
     "llama",
+    "qianwan",
+    "alibaba",
+    "mistral",
+    "zhipu",
+    "KEG lab",
+    "01.AI",
+    "AI2",
+    "Tülu",
     "NETWORK ERROR DUE TO HIGH TRAFFIC. PLEASE REGENERATE OR REFRESH THIS PAGE.",
     "$MODERATION$ YOUR INPUT VIOLATES OUR CONTENT MODERATION GUIDELINES.",
-    "API REQUEST ERROR. Please increase the number of max tokens.",
```

Comment:
I also saw this error in the log file. Should we keep this?

Reply:
I think the exact message is as follows
<img width="504" alt="image" src="https://github.com/lm-sys/FastChat/assets/1206058/3c5babea-b7fd-4c81-b27f-e44cd6e83387">
but let's keep it to be conservative.